src/components/PortfolioDetailItem/PortfolioDetailItem.tsx

```diff
@@ -0,0 +1,74 @@
+import {useMemo} from 'react';
+import * as S from './PortfolioDetailItem.style';
+
+function PortfolioDetailItem({
+  coin: {quantity, averagePrice, trade_price, korean_name, english_name},
+}: PortfolioDetailItemProp) {
+  const purchaseAmount = useMemo(() => quantity * averagePrice, [quantity, averagePrice]);
+  const evaluationAmount = useMemo(() => quantity * trade_price, [quantity, trade_price]);
+  const valuationGainsAndLosses = useMemo(() => evaluationAmount - purchaseAmount, [evaluationAmount, purchaseAmount]);
+  const rateOfGainOrLossOnValuation = useMemo(
+    () => (valuationGainsAndLosses / purchaseAmount) * 100,
+    [valuationGainsAndLosses, purchaseAmount]
+  );
+
+  const headerItemList = useMemo(
+    () => [
+      {title: '평가손익', value: valuationGainsAndLosses, formatted: valuationGainsAndLosses.toLocaleString()},
+      {title: '수익률', value: rateOfGainOrLossOnValuation, formatted: `${rateOfGainOrLossOnValuation.toFixed(2)}%`},
+    ],
+    [valuationGainsAndLosses, rateOfGainOrLossOnValuation]
+  );
+
+  const bodyItemList = useMemo(
+    () => [
+      [
+        {title: '보유수량', value: quantity.toLocaleString(), unit: english_name},
+        {title: '매수평균가', value: averagePrice.toLocaleString(), unit: 'KRW'},
+      ],
+      [
+        {title: '평가금액', value: evaluationAmount.toLocaleString(), unit: 'KRW'},
+        {title: '매수금액', value: purchaseAmount.toLocaleString(), unit: 'KRW'},
+      ],
+    ],
+    [quantity, english_name, averagePrice, evaluationAmount, purchaseAmount]
+  );
+
+  return (
+    <S.PortfolioDetail>
+      <S.Header>
+        <S.CoinName>
+          {korean_name}
+          <br />({english_name})
+        </S.CoinName>
```

Comment:
```suggestion
        <S.CoinName>{korean_name + '\n' + english_name}</S.CoinName>
```
그냥 <br /> 태그는 semantic에 맞지 않는 듯 하여 이렇게 하고서 css white-space: pre-line로 수정해보면 어떨까요??

Reply:
수정완료했어요~ 감사합니다ㅎㅎ

Reply:
정말 괜찮은 방법이네요! 가독성도 한층 올라간 것 같아요